### PR TITLE
feat: add docs about auctions and protocol tokens

### PR DESCRIPTION
**Pull request by mootz12**

### CODE REVIEW

File: tech-docs/core-contracts/lending-pool/protocol-tokens.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Protocol Tokens
```
Comment (markuspluna) [inline tech-docs/core-contracts/lending-pool/protocol-tokens.md:1]: We should call out these are purely accounting structures. Calling this Protocol Accounting may be a better option

Comment (markuspluna) [reply tech-docs/core-contracts/lending-pool/protocol-tokens.md:1]: Yeah I think just use the word token less, tokenized should be cut entirely

File: users/auctions.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+This results in a few key blocks for auctions. At block 0, the bidder pays the full bid but receives no assets. At block 200, the bid and lot are exactly the base value of the auction. At block 400, the bidder pays nothing but receives the full lot. The TL;DR is that in the first 200 blocks, the lot increases by 0.5% of the base per block, and the final 200 blocks, the bid decreases by 0.5% of the base per block.
+
+### Interest Auctions
+
+Interest auctions facilitate the sale of the backstop's earned interest for backstopping a pool in exchange for backstop tokens. These backstop tokens are then donated to the backstop such that they are shared among all backstop depositors.
+
+#### Bid
+
+The bid contains only the backstop token. On a successful fill, the backstop token is transferred from the bidder to the pool.
```
Comment (markuspluna) [inline users/auctions.md]: Add link that explains what a backstop token is (or a description)

**Review by markuspluna — CHANGES_REQUESTED** [review]
Few small changes needed

File: pool-creators/selecting-an-oracle.md [hunk 3]
```diff
@@ -2,11 +2,54 @@
...
 
-Oracles must conform to the SEP-40 oracle standard\
+**Oracles CANNOT be changed after a pool is created, so please be very careful when selecting an oracle for a pool.**
+
+Oracles must support the "lastprice" and "decimals" functions on the SEP-40 oracle standard\
 [https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0040.md](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0040.md)
 
+### Test the Oracle
+
+It is required to verify that "lastprice" works for all potential reserves the pool will maintain. Please note that the Blend pool will always invoke the oracle with `Stellar::Asset({contract_id})` to fetch last price.
```
Comment (Ryang-21) [inline pool-creators/selecting-an-oracle.md]: is it worth noting that `contract_id` is an `Address` object?

Comment (drew-boy) [inline pool-creators/selecting-an-oracle.md]: take out 'that' in 'please note that...'

File: pool-creators/selecting-an-oracle.md [hunk 4]
```diff
@@ -2,11 +2,54 @@
...
+const assets = ["C...", "C...", "C..."];
+
+const oracle = await PoolOracle.load(network, oracle_id, assets);
+// validate results
+console.log(JSON.stringify(oracle, replacer, 2));
+```
+
 ### Oracle Adapters
 
 When a generalized oracle is insufficient for a pool, i.e. it doesn't support all assets the pool needs prices for, the pool creator may need to use an oracle adaptor. An oracle adaptor is a custom oracle contract that uses custom logic to aggregate multiple oracle feeds or impose desired behavior, such as reporting TWAP prices.
```
Comment (drew-boy) [inline pool-creators/selecting-an-oracle.md]: ", i.e. it doesn't support all assets the pool needs prices for," -> (i.e. it doesn't support all assets the pool needs prices for)

**Review by markuspluna — APPROVED** [review]

**Pull request merged**